Data/market-hours/market-hours-database.json

```diff
@@ -37956,166 +37927,1451 @@
         "12/25/2023": "17:00:00"
       }
     },
-    "Future-nymex-RB": {
+    "Future-cme-FT1": {
       "dataTimeZone": "UTC",
-      "exchangeTimeZone": "America/New_York",
+      "exchangeTimeZone": "America/Chicago",
       "sunday": [
         {
-          "start": "18:00:00",
+          "start": "17:00:00",
           "end": "1.00:00:00",
           "state": "premarket"
         }
       ],
       "monday": [
         {
           "start": "00:00:00",
-          "end": "09:30:00",
+          "end": "08:30:00",
           "state": "premarket"
         },
         {
-          "start": "09:30:00",
-          "end": "17:00:00",
+          "start": "08:30:00",
+          "end": "16:00:00",
           "state": "market"
         },
         {
-          "start": "18:00:00",
+          "start": "17:00:00",
           "end": "1.00:00:00",
           "state": "postmarket"
         }
       ],
       "tuesday": [
         {
           "start": "00:00:00",
-          "end": "09:30:00",
+          "end": "08:30:00",
           "state": "premarket"
         },
         {
-          "start": "09:30:00",
-          "end": "17:00:00",
+          "start": "08:30:00",
+          "end": "16:00:00",
           "state": "market"
         },
         {
-          "start": "18:00:00",
+          "start": "17:00:00",
           "end": "1.00:00:00",
           "state": "postmarket"
         }
       ],
       "wednesday": [
         {
           "start": "00:00:00",
-          "end": "09:30:00",
+          "end": "08:30:00",
           "state": "premarket"
         },
         {
-          "start": "09:30:00",
-          "end": "17:00:00",
+          "start": "08:30:00",
+          "end": "16:00:00",
           "state": "market"
         },
         {
-          "start": "18:00:00",
+          "start": "17:00:00",
           "end": "1.00:00:00",
           "state": "postmarket"
         }
       ],
       "thursday": [
         {
           "start": "00:00:00",
-          "end": "09:30:00",
+          "end": "08:30:00",
           "state": "premarket"
         },
         {
-          "start": "09:30:00",
-          "end": "17:00:00",
+          "start": "08:30:00",
+          "end": "16:00:00",
           "state": "market"
         },
         {
-          "start": "18:00:00",
+          "start": "17:00:00",
           "end": "1.00:00:00",
           "state": "postmarket"
         }
       ],
       "friday": [
         {
           "start": "00:00:00",
-          "end": "09:30:00",
+          "end": "08:30:00",
           "state": "premarket"
         },
         {
-          "start": "09:30:00",
-          "end": "17:00:00",
+          "start": "08:30:00",
+          "end": "16:00:00",
           "state": "market"
         }
       ],
       "saturday": [],
-      "holidays": [
-        "4/3/2015",
-        "4/2/2021"
-      ],
+      "holidays": [],
```

Comment:
Never mind, I've just realized that the holidays for cme futures of class _Equities_ are described in the general entry (`Futures-cme-[*]`) since they are common for all cme futures